info.go

```diff
@@ -37,7 +37,6 @@ func (s *Server) HandleInfoRequest(w *Response, r *http.Request) *InfoRequest {
 	// load access data
 	ret.AccessData, err = w.Storage.LoadAccess(ctx, ret.Code)
 	if err != nil {
-		s.setErrorAndLog(w, E_INVALID_REQUEST, err, "handle_info_request=%s, code=%s", "failed to load access data", ret.Code)
```

Comment:
instead of delete this line, please put condition inside setErrorAndLog function if internalError = IgnorePrintError(please check this in oauthapp module)

Reply:
okay ko, revalidate